stacks/apps/dovetail-router.yml

```diff
@@ -172,6 +173,25 @@ Resources:
           SetIdentifier: !Ref AWS::StackName
           Type: A
 
+  VanityRecordSetGroup:
```

Comment:
Do you imagine these being used for anything other than a human accessing a specific region for testing?

Reply:
I could see using them to get multi-region DTR off the ground.  Or even for monitoring/alarms.

Reply:
I would prefer to use the existing DNS entries for the regional Dovetail ALBs for those sorts of things, since they already exist, and there's no risk of them drifting or changing over time (or when they do it's explicitly because the underlying resource changed). It's also more consistent with how we handle the other ALBs. I'm fine with having some convenience/vanity domains to make devs' lives easier, but I don't want them to sneak into critical paths.

Reply:
Yeah, fair enough.  So my first use would really be devtools - the various DTR debugging scripts, and `awssh` through the jump server stuff.